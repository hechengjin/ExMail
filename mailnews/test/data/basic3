Content-Type: text/html; charset=iso-8859-1

<html><body><h1>The HTML tags here <b>are</b> supposed to be stripped out.</h1></body></html>
