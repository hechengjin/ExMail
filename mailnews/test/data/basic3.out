The HTML tags here are supposed to be stripped out.